docs/modules/PercentHealth.md

```diff
@@ -20,7 +21,7 @@ objectives:
   scaling: 5
 ```
 
-`update period` - how often to update player's health (in ticks)  
+`update period` - how often to forcibly update all online player's health (in ticks)
```

Comment:
If we're going to increase the force refresh time to 5 seconds from 1 we should probably add a note to say that for plugins that set health manually without damaging the player a lot the number should be lowered to keep it responsive